### DISCUSSION TITLE
使用@antv/g6-extension-react的ReactNode使用antd UI元素图中node显示不全

### DISCUSSION THREAD

**Discussion by tqq1994516** [antvis/G6#6109] [Q&A]
![image](https://github.com/user-attachments/assets/19ad4610-6269-400b-a589-29e431a04e42)
![image](https://github.com/user-attachments/assets/86e521ab-ca6b-4fee-8d04-9236c76da999)
node初始位置在左上角且显示不全，拖拽至图中或设置autoFit属性依旧无法正常显示

**Comment by Aarebecca** [comment]
这个问题目前正在解决中

**Reply by tqq1994516** [reply]
期待
